data/json/vehicle_parts.json

```diff
@@ -3746,6 +3746,26 @@
         ]
     },{
         "type" : "vehicle_part",
+        "id" : "mounted_m134",
+        "name" : "mounted M134D-H Minigun",
+        "symbol" : "t",
+        "color" : "green",
+        "broken_symbol" : "#",
+        "broken_color" : "green",
+        "damage_modifier" : 80,
+        "durability" : 400,
+        "range" : 12,
+        "fuel_type" : "308",
```

Comment:
This entry is not required fro turrets anymore. The ammo type is taken directly from the gun item.